pkg/k8s-client/client.go

```diff
@@ -78,30 +75,30 @@ func (c *client) SetAnnotationsOnPod(pod *kapi.Pod, annotations map[string]strin
 	}
 
 	podDesc := pod.Namespace + "/" + pod.Name
-	glog.Infof("Setting annotations %v on pod %s", annotations, podDesc)
+	log.Debug().Msgf("Setting annotations %v on pod %s", annotations, podDesc)
 	patchData, err = json.Marshal(&patch)
 	if err != nil {
-		glog.Errorf("Error in setting annotations on pod %s: %v", podDesc, err)
+		err = fmt.Errorf("failed to set annotations on pod %s: %v", podDesc, err)
 		return err
 	}
 	return c.PatchPod(pod, types.MergePatchType, patchData)
 }
 
 // PatchPod applies the patch changes
 func (c *client) PatchPod(pod *kapi.Pod, patchType types.PatchType, patchData []byte) error {
-	glog.V(3).Infof("PatchPod(): namespace: %s, podName: %s", pod.Namespace, pod.Name)
+	log.Debug().Msgf("PatchPod(): namespace: %s, podName: %s", pod.Namespace, pod.Name)
```

Comment:
maybe print patch data as string as well? unless the data is expected to be big and then thats good enough

Reply:
I think it big, let keep it as is